src/common/EssenceType.cpp

```diff
@@ -117,6 +117,8 @@ static const EssenceTypeInfo ESSENCE_TYPE_INFO[] =
     {RDD36_4444_XQ,             PICTURE_ESSENCE,        "RDD36 4:4:4:4 XQ",                     "RDD36_4444_XQ"},
     {JPEG2000_CDCI,             PICTURE_ESSENCE,        "JPEG 2000 CDCI",                       "JPEG2000_CDCI"},
     {JPEG2000_RGBA,             PICTURE_ESSENCE,        "JPEG 2000 RGBA",                       "JPEG2000_RGBA"},
+	{JPEGXS_CDCI,               PICTURE_ESSENCE,        "JPEG XS CDCI",                         "JPEGXS_CDCI"},
+	{JPEGXS_RGBA,               PICTURE_ESSENCE,        "JPEG XS RGBA",                         "JPEGXS_RGBA"},
```

Comment:
Could you also add entries to the XML enum around [here](https://github.com/bbc/bmx/blob/main/meta/mxf2raw/mxf2raw_info.xsd#L981)?